Makefile

```diff
@@ -153,6 +154,17 @@ install:
 install-statik:
 	go get -u github.com/rakyll/statik
 
+.PHONY: install-go-bindata-assetfs
+install-go-bindata-assetfs:
+	go get -u github.com/jteeuwen/go-bindata/...
+	go get -u github.com/elazarl/go-bindata-assetfs/...
```

Comment:
can we not use `rakyll/statik` above? I would prefer to have only one method of embedding.

Reply:
Not possible at the moment bc it cannot be used to host multiple statik files and we already use it https://github.com/rakyll/statik/issues/56

Reply:
As it is only one file, could we just embed the content in a raw string? Would also prefer not to have a separate approach - and if statik eventually supports hosting multiple files then we could look at that again.

Reply:
There are two index mapping files. The whole purpose of this is to migrate from string variable as it is not possible to consume it from other places than golang.

Reply:
@yurishkuro I think we were using `go-bindata` in jaeger and it was switched to statik. Do you know why? 

We can use only one lib if it is a problem I am keen on handling it in a separate PR.

Reply:
The reason I liked statik is because it's a single library for both generator and runtime, instead of two separate libs. But I don't have a strong objection to go back to bindata, as long as we remove the dependency on statik (i.e. let's use only one mechanism for embedding).

Reply:
There are a lot of libs doing this https://tech.townsourced.com/post/embedding-static-files-in-go. I can have a look and find one which works the best for us. I will do it in a separate PR.